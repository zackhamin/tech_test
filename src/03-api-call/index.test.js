const fetchLukeSkywalker = require('.');
const axios = require("axios");

describe('fetchLukeSkywalker', () => {
  it('returns correct information about Luke Skywalker', async () => {
    const actualResult = await fetchLukeSkywalker();

    expect(actualResult).toMatchSnapshot();
  });
});
