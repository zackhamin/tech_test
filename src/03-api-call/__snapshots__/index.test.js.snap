// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchLukeSkywalker returns correct information about Luke Skywalker 1`] = `
 {
  "birth_year": "19BBY",
  "created": "2014-12-09T13:50:51.644000Z",
  "edited": "2014-12-20T21:17:56.891000Z",
  "eye_color": "blue",
  "films": Array [
    "http://swapi.dev/api/films/1/",
    "http://swapi.dev/api/films/2/",
    "http://swapi.dev/api/films/3/",
    "http://swapi.dev/api/films/6/",
  ],
  "gender": "male",
  "hair_color": "blond",
  "height": "172",
  "homeworld": "http://swapi.dev/api/planets/1/",
  "mass": "77",
  "name": "Luke Skywalker",
  "skin_color": "fair",
  "species": Array [],
  "starships": Array [
    "http://swapi.dev/api/starships/12/",
    "http://swapi.dev/api/starships/22/",
  ],
  "url": "http://swapi.dev/api/people/1/",
  "vehicles": Array [
    "http://swapi.dev/api/vehicles/14/",
    "http://swapi.dev/api/vehicles/30/",
  ],
}
`;
